package.json

```diff
@@ -49,7 +47,7 @@
     "@babel/preset-react": "^7.18.6",
     "babel-loader": "^9.1.0",
     "css-loader": "^6.7.1",
-    "eslint": "8.22.0",
+    "eslint": "^8.22.0",
```

Comment:
I believe that caret symbol was not added under an assumption that the linter behaviour might change in a newer version. We can refer to this section to understand the use of the caret (`^`) symbol: https://github.com/npm/node-semver?tab=readme-ov-file#caret-ranges-123-025-004

Reply:
Sure. It is rare that ESLint extension in VS Code automatically changes this for me.